src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const { peopleSlug } = useParams();
+
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+  } = person;
+
+  const currentRow = slug === peopleSlug;
+  const isWoman = person.sex === 'f';
+  const isMother = motherName || '-';
+  const isFather = fatherName || '-';
```

Comment:
isMother and isFather are not boolean values, so it's better to find a different name for them